src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -275,6 +294,19 @@ public function denormalize($data, $class, $format = null, array $context = [])
             }
 
             $value = $this->validateAndDenormalize($class, $attribute, $value, $format, $context);
+
+            foreach ($attributesMetadata as $currentAttribute => $metadata) {
+                if (empty($metadata->getEmbedProperties())) {
```

Comment:
Same here, store the return value in a variable.

Reply:
Done